client/components/plans/plan-price/style.scss

```diff
@@ -1,32 +1,46 @@
-.plan-header .plan-price {
+.plan-header .plan-price,
```

Comment:
Duplicating classes like this will make the CSS really bloated. Can we use a mixin instead?